dotnet/src/webdriver/Firefox/FirefoxDriver.cs

```diff
@@ -178,6 +180,28 @@ public override IFileDetector FileDetector
             set { }
         }
 
+        /// <summary>
+        /// Sets the command context used when issuing commands to geckodriver.
+        /// </summary>
+        /// <exception cref="WebDriverException">If response is not recognized</exception>
+        /// <returns>The context of commands.</returns>
+        public FirefoxCommandContext GetContext()
+        {
+            string response = this.Execute(GetContextCommand, null).Value.ToString();
+
+            if (response == "Content")
```

Comment:
A better approach would be (`true` in the below is a case-insensitive comparison):
```csharp
FirefoxCommandContext output;
bool success = Enum.TryParse<FirefoxCommandContext>(response, true, out output);
if (!success)
{
    throw new WebDriverException(string.Format(CultureInfo.InvariantCulture, "Do not recognize response: {0}; expected Context or Chrome";
}

return output;
}
```

Reply:
Aha, there is a good way to parse the enum, nice.